safe-global/safe-wallet-web
# POC: Skip safe creation status screen

DO NOT MERGE

## What it solves

Resolves #3612 

## How this PR fixes it

- Removes the status screen from safe creation
- Waits for the wallet submission before leaving the safe creation screen
- Navigates the user to their dashboard as soon as the safe creation is pending
- Disables all transaction buttons for n/m safe configs and shows an "Activate Safe" Button in the sidebar instead

## How to test it

1. Create a n/m safe
2. Observe staying on the review screen until you submit the transaction in your wallet
4. Observe the dashboard with a loading screen

## Screenshots
![Screenshot 2024-05-15 at 14 14 05](https://github.com/safe-global/safe-wallet-web/assets/5880855/76777d95-9c27-4095-9d3a-54ae2c8e8f85)

## Checklist
